docs/evaluate/how-it-works.mdx

```diff
@@ -0,0 +1,90 @@
+---
+id: how-it-works
+title: How Temporal Works
+sidebar_label: How It Works
+description: This guide provides a short overview of How Temporal Works.
+toc_max_heading_level: 4
+keywords:
+  - temporal
+  - evaluate-temporal
+  - how-temporal-works
+tags:
+  - temporal
+  - evaluate-temporal
+  - how-temporal-works
+---
+Temporal offers an entirely new way to build scalable and reliable applications.
+
+## Build Invincible Apps 
+
+Temporal ensures your applications are reliable by default, handling everything from minor network outages to significant issues like unexpected reboots or hardware failures.
+By offloading the responsibility of failure management from the application to the platform, this removes the need for extensive coding, testing, and maintenance tasks.
+
+This shift simplifies the development process and enables your application to seamlessly recover from disruptions.
+It does so by automatically reconstructing its previous state.
+Your application logic can be recovered, replayed, or paused from any previous point in the application process.
+
+![Describing a Temporal Application - How It Works](/img/how-it-works/hiw-yourapplicationclient.png)
+
+### Temporal Application
+
+A Temporal application is the code you write, comprised of Workflow and Activities. 
+A Temporal Workflow defines your overall business logic. That business logic might involve moving money between bank accounts, processing orders, deploying cloud infrastructure, training an AI model, or something else entirely.
+The Worker runs the Workflow and Activity code.
+
+![Workflow Function Execution - How It Works](/img/how-it-works/hiw-workflowexecution.png)
+
+### Activity
+Activities represent code (either a functions or method) that handle tasks prone to errors or failure, such as network requests, calling a service, sending an email, or generating random numbers.
```

Comment:
I think that we should explain why code prone to failure/non-deterministic code is encapsulated in Activities. Reader might be confused why that is. The reason is because we can retry them over and over, making Temporal resilient against transient issues. I think that's super important to mention the retry policy.